ui/Screen/Org/Fund.svelte

```diff
@@ -1,21 +1,15 @@
 <script>
-  import { transactions as store } from "../../src/transaction.ts";
+  import { transactions as txStore } from "../../src/transaction.ts";
+  import { org, orgTransactions } from "../../src/org.ts";
   import { Wallet, Remote } from "../../DesignSystem/Component";
 
   export let params = null;
-
-  const orgTransactions = transactions => {
-    return transactions.filter(tx => {
-      return "domainId" in tx.messages[0]
-        ? tx.messages[0].domainId == params.id
-        : null;
-    });
-  };
+  const accountId = $org.data.accountId;
 </script>
 
-<Remote {store} let:data={transactions}>
+<Remote store={txStore} let:data={transactions}>
   <Wallet
-    transactions={orgTransactions(transactions)}
-    balance="3484"
-    address="hyda4fhdx8up8hhocagkdz3d41txb98stw4pkqe3uommo1p5m6s9oy" />
+    transactions={orgTransactions(transactions, params.id)}
+    balance={3484}
+    address={accountId} />
```

Comment:
Should we adjust the param of the `Wallet` component if we now refer to it as account id?

Reply:
yeah we should uniformalize the terminologies. An address, technically speaking, is the account id encoded in a specific way. We use the SS58 format as the address. The `accountId` value here is actually the address-encoded-like representation of the entity's account id, so we should just have it called `address` in throughout the whole UI. since we might need to do this in other places, I'd be ok with sticking to this for now and then make that change in one go in a follow up PR.

Reply:
I think the `params.id` is actually the orgId, not the accountId

Reply:
ok

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/691/commits/be8cf18abc7416c9281e19c1a5e92c062bfa71a9